docs/kqueen.rst

```diff
@@ -152,6 +152,142 @@ in the configuration file, set the environment variable matching the KQUEEN_<con
       - 127.0.0.0/8
       - Addresses allowed to access metrics endpoint without token
 
+Before you provision a Kubernetes cluster, you may need to deploy and configure the following external services:
+
+External services deployment
+-------
+* **Members managing** - To provide user registration by invitations from other users by emails, mail server should be set up.
+It is possible to use kqueen predefined mail service or to run new one.
+
+* **Metrics collecting** - Its possible to add external Prometheus serveri to extend monitoring in Kqueen. In case of using external server, need to include rules from kqueen/prod/prometheus into existing Prometheus service. Otherwise, its possible to use predefined Prometheus, defined in docker-compose.production.yml.
```

Comment:
serveri -> server

Reply:
in `docker-compose.production.yml <https://github.com/Mirantis/kqueen/blob/master/docker-compose.production.yml>`_